packages/lib/src/vis/scatter/ScatterPoints.tsx

```diff
@@ -88,6 +90,15 @@ function ScatterPoints(props: Props) {
     invalidate();
   }, [color, dataGeometry, invalidate, position]);
 
+  const zoomVector = useThree((state) => state.camera.scale);
+
+  useFrame(() => {
+    const zoom =
+      Math.sqrt(zoomVector.x * zoomVector.x + zoomVector.y * zoomVector.y) /
```

Comment:
Isn't this just `zoomVector.length()`? On the other hand, you could just take `zoomVector.x` as the threshold, since axial zoom is disabled (and doesn't make sense, since we can't have oval points and click targets).